Lib/_pyrepl/reader.py

```diff
@@ -421,42 +421,15 @@ def calc_screen(self) -> list[str]:
 
     @staticmethod
     def process_prompt(prompt: str) -> tuple[str, int]:
```

Comment:
I may be missing something but looks like the old function was tracking nested ANSI escape seas outside `\x01` and `\x02` brackets but this one drops this no? What is the rationale?

Reply:
There are three things the old function was trying to do (badly):
- ignore anything between \001 and \002 for length calculation; and
- remove \001 and \002 characters from the emitted prompt; and
- also ignore any ANSI escape sequences from length calculation.

Since wlen() is already doing 3., a contributor here needed to recreate this function to not do that, because we were doing that later anyway. But it's simpler to just use the main wlen() with a string stripped of \001 .. \002 bracketed content.

I find the new form much easier on the eyes.